docs/how-to-edit-content.md

```diff
@@ -0,0 +1,127 @@
+
+![Create your own branch](images/create-branch.png)
+
+Once created, you'll automatically be switched from the master branch to your new branch. Because you created your branch while being on the master branch prior, the code in your new branch should be identical to those in the master branch.
+
+---
+
+### Perform your page edits
+Now that you've created your own branch, you can make all of your edits.
+
+#### 1. First, locate the page to be edited.
+
+For example, to edit `www.va.gov/disability/eligibility/`, we would navigate to [`/pages/disability/eligibility.md`](https://github.com/department-of-veterans-affairs/vagov-content/blob/awesome-new-branch/pages/disability/eligibility.md).
+
+#### 2. Click the pencil icon in the top-right corner of the panel containing the page contents to open an editor.
+
+![Click the pencil icon to open an editor](images/create-branch.png)
+
+An editor like the one below should open -
+
+![Page editor](images/page-editor.png)
+
+__Pages are written using Markdown, whi
+
+#### 3. Finally, save your changes.
+Once you've made your changes, scroll to the bottom of the page. Once there, you should see a a box titled "Commit changes". On GitHub, this means to save your changes to your branch. To do so, just type a title along with an optional description, then press the green "Commit changes" button.
+
+![Commit changes](images/save-edits.png)
+
+#### 4. Repeat for any additional edits
+If there are any related edits that need to be made along with this, you can make those changes on the same branch. To do so, just confirm that the Branch dropdown in the top-corner reflects the name of your branch, and repeat the steps above.
+
+---
+
+### Open a pull request
+A [pull request](https://help.github.com/articles/about-pull-requests/) is the term used to describe proposing the changes of your branch to become part of the master branch. You can think of the changes in your branch as drafts, and once moved into master via a pull request, those changes are published. To open a pull request, following these steps.
+
+#### 1. Click the "Pull requests" tab along the top of the page
+
+![Click pull requests tab](images/pr-tab.png)
+
+#### 2. Click the "New pull request" button in the top-right corner
+
+![New pull request](images/new-pr.png)
+
+#### 3. Select your branch for comparison with the master branch
+
+![Select your branch](images/pr-compare.png)
+
+#### 4. Click the "Create pull request" button
+
+![Create pull request](images/create-pr.png)
+
+#### 5. Optionally, edit the title/description of your pull request and click "Create pull request"
+
+![Edit pull request title](images/edit-pr-title.png)
+
+Your pull request is created!
```

Comment:
Is it worth mentioning the Heroku instances that people can use to view their change?